src/qt/coincontroldialog.cpp

```diff
@@ -55,7 +55,7 @@ CoinControlDialog::CoinControlDialog(CCoinControl& coin_control, WalletModel* _m
     contextMenu->addAction(tr("&Copy address"), this, &CoinControlDialog::copyAddress);
     contextMenu->addAction(tr("Copy &label"), this, &CoinControlDialog::copyLabel);
     contextMenu->addAction(tr("Copy &amount"), this, &CoinControlDialog::copyAmount);
-    copyTransactionHashAction = contextMenu->addAction(tr("Copy transaction &ID"), this, &CoinControlDialog::copyTransactionHash);
+    m_copy_transaction_outpoint_action = contextMenu->addAction(tr("Copy transaction &ID and output index"), this, &CoinControlDialog::copyTransactionOutpoint);
```

Comment:
Anything against having both actions?

Reply:
@promag seems a bit more GUI clutter for no purpose. Its a manual action touching the clipboard so it can't interfere with any automation, so the user can always just manually delete the `:n` on the end if they don't want it. IMO txid alone is useless in this situation because this is a specific outpoint selection dialog.